components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -914,8 +912,8 @@ public String getOpenIDConnectIDTokenExpiration() {
      *
      * @return ID Token expiry time in milliseconds.
      */
-    public long getOpenIDConnectIDTokenExpiryTimeInMillis() {
-        return openIDConnectIDTokenExpiryTimeInMillis;
+    public long getOpenIDConnectIDTokenExpiryTimeInSeconds() {
```

Comment:
Already have one API (getOpenIDConnectIDTokenExpiration) to get the value in seconds. Can't we keep `getOpenIDConnectIDTokenExpiryTimeInMillis` and return `openIDConnectIDTokenExpiryTimeInSeconds * 1000L`

Reply:
I was thinking the same. Since the other expiry times in seconds are returned as int wanted to maintain consistency...

getOpenIDConnectIDTokenExpiration returns the value as a String